Parkinsons-Mouse-tutorial.ipynb

```diff
@@ -0,0 +1,3062 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Alpha Rarefaction and Selecting a Rarefaction Depth"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now have a feature table (observation matrix) of ASVs in each sample, and a phylogenetic tree representing those ASVs, so are almost ready to perform various analyses of microbial diversity. However, first we must normalize our data to account for uneven sequencing depth between samples."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Although sequencing depth in a microbiome sample does not directly relate to the original biomass in a community, the relative sequencing depth has a large impact on observed communities [(Weiss et al, 2017)](https://pubmed.ncbi.nlm.nih.gov/28253908/). Therefore, for most diversity metrics, a normalization approach is needed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Current best practices suggest the use of rarefaction, a normalization via sub-sampling without replacement. Rarefaction occurs in two steps: first, samples which are below the rarefaction depth are filtered out of the feature table. Then, all remaining samples are subsampled without replacement to get to the specified sequencing depth. It’s both important and sometimes challenging to select a rarefaction depth for diversity analyses. Several strategies exist to figure out an appropriate rarefaction depth - we will primarily consider alpha rarefaction in this tutorial, because it is a data-driven way to approach the problem."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We’ll use `diversity.visualizers.alpha_rarefaction` to subsample the ASV table at different depths (between `min_depth` and `max_depth`) and calculate the alpha diversity using one or more metrics . When we checked the feature table, we found that the sample with the fewest sequences in the denoised table has 85 features and that the sample with the most has 4996 features. We want to set a maximum depth close to the maximum number of sequences. We also know that if we look at a sequencing depth around 4250 sequences per sample, we’ll be looking at information from 22 samples. So, let’s set this as our maximum sequencing depth.\n",
+    "\n",
+    "At each sampling depth, 10 rarefied tables are usually calculated to provide an error estimate, although this can be adjusted using the `--p-iterations` parameter. We can check and see if there is a relationship between the alpha diversity and metadata by specifying the metadata file for the `metadata = sample_metadata` parameter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "normalized_data = diversity.visualizers.alpha_rarefaction(denoised_sequences.table, \n",
```

Comment:
Dlaczego jest normalized_data i normalized_data_1? chyba wystarczy tylko to drugie